api/test/dbTest.js

```diff
@@ -0,0 +1,19 @@
+require('dotenv').config({path: "../../.env.test"});
```

Comment:
This file won't exist on any machines (unless a developer creates it). Eventually this test will need to be updated since we'll want to allow setting test specific environment variables . . . but for now, just don't create the file and the tests should pass.